themes/bootstrap5/js/account_ajax.js

```diff
@@ -79,10 +96,19 @@ VuFind.register('account', function Account() {
     }
   };
 
+  /**
+   * Get status as a number for module from _statuses object
```

Comment:
Should we add something like "(see constants defined above -- LOADING, MISSING, INACTIVE)"